next-client/src/components/claim/Claim.tsx

```diff
@@ -64,3 +77,49 @@ export function QuoteIcon({ num }: { num: number }) {
     </Row>
   );
 }
+
+/**
+ * Bridging score badge showing the claim's bridging potential.
+ * Shows for non-neutral scores (score < 0 or score >= 0.3).
+ * When sortByBridging is enabled, shows all scores regardless of threshold.
+ * Uses simple labels: "Bridging" (green), "Divisive" (red), or "Neutral" (gray).
+ */
+function BridgingBadge({
+  score,
+  forceShow = false,
+}: {
+  score: number;
+  forceShow?: boolean;
+}) {
+  // Don't show badge for neutral scores (between 0 and 0.3) unless forceShow
+  if (!forceShow && score >= 0.0 && score < 0.3) {
+    return null;
+  }
```

Comment:
I think it makes more sense to move the isDevisive and isHighBridging aboe this and check against them
```suggestion
  if (!forceShow && !isDevisive && !isBridging) {
    return null;
  }
```